client/state/current-user/selectors.js

```diff
@@ -44,6 +44,21 @@ export function getCurrentUserLocale( state ) {
 }
 
 /**
+ * Returns the isRTL value for the current user.
+ *
+ * @param  {Object} state  Global state tree
+ * @return {?Bool}  Current user is RTL
+ */
+export function getCurrentUserIsRTL( state ) {
```

Comment:
Oh yes, that's a good call: it's recommended that we introduce all new selectors there. More about that can be found here: https://github.com/Automattic/wp-calypso/blob/master/client/state/selectors/README.md